include/openssl/evp.h

```diff
@@ -695,6 +695,12 @@ const EVP_MD *EVP_sha224(void);
 const EVP_MD *EVP_sha256(void);
 const EVP_MD *EVP_sha384(void);
 const EVP_MD *EVP_sha512(void);
+const EVP_MD *EVP_sha3_224(void);
+const EVP_MD *EVP_sha3_256(void);
+const EVP_MD *EVP_sha3_384(void);
+const EVP_MD *EVP_sha3_512(void);
+const EVP_MD *EVP_shake128(void);
+const EVP_MD *EVP_shake256(void);
```

Comment:
They *are* implemented. But not fully "wired" into the framework. The reason for the latter is that it's impossible to use them right now. Next planned step would be to add digest flag denoting extensible-output capability, and ctrl that allows to choose output length. It would be appropriate to document and unblock them *then*.